core/templates/services/loader.service.ts

```diff
@@ -34,22 +32,21 @@ export class LoaderService {
    * In order to keep the variables same, static is used until migration is
    * complete.
    */
-  static loadingMessageSubject: Subject<string> = new Subject<string>();
+  static loadingMessageChangedEventEmitter = new EventEmitter<string>();
+  get onLoadingMessageChange(): EventEmitter<string> {
```

Comment:
This is an interesting change! My first reaction was that the semantics are still a bit off and we should be using regular functions here. But on reading the callsites I really like it and the removal of the "()" does make a difference.

So yeah, if you like it too, let's go with this as a convention. (Is it standard -- do other codebases use it?) In any case, if this is to be a convention, you'll need to fix this for the other event emitter too.

Reply:
Changed the other emitter too.

Reply:
> Is it standard -- do other codebases use it?

I am not really sure if this is a standard but I have seen other codebases using this sort of semantics.

Reply:
Which other codebases? Please link to examples, for reference. (In general, try to always support your statements with evidence.)

Reply:
Exact usecase: @ngx-translate uses this sort of convention. Angulars codebase matches our use-case partially. They use `get` but for variables inside a class. EventEmitter is also a variable in our class so partially matches our use case.
ngx-translate: https://github.com/ngx-translate/core/blob/master/projects/ngx-translate/core/src/lib/translate.service.ts#L71-L79